.github/workflows/test.yaml

```diff
@@ -6,12 +6,18 @@ on:
 
 jobs:
   Test:
-    name: Test on Erlang/OTP ${{ matrix.otp_version }} and ${{ matrix.os }}
+    name: Test on Erlang/OTP ${{ matrix.beam_versions.otp }}, Elixir ${{ matrix.beam_versions.elixir }} and ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
 
     strategy:
       matrix:
-        otp_version: [23, 24, '25.0-rc3']
+        beam_versions:
+          - otp: 23
+            elixir: '1.11.4'
+          - otp: 24
+            elixir: '1.13.4'
+          # - otp: '25.0-rc3'
+          #   elixir: '1.13.4'
```

Comment:
The installation when using this combination errors out. Ideally, this section in the matrix would skip all Elixir steps and only run the Erlang parts.